### PR TITLE
Add line output

### PR DESCRIPTION
**Pull request by littlelion308**
Added properties for skipRules and ruleConfig, switch output to file to cli output to get more information out of pointers, and added lines output.

### CODE REVIEW

File: src/main/java/com/ethlo/zally/ZallyRunner.java [hunk 1]
```diff
@@ -180,11 +183,16 @@ private Object createInstance(Class<?> type)
...
-    private Result handleViolation(final CheckDetails details, Violation violation)
+    private Result handleViolation(String url, final CheckDetails details, Violation violation)
     {
-        // TODO: Handle pointers better to make it easier to know where the error is
-        //final JsonPointer pointer = violation.getPointer();
-        //System.out.println(pointer.toString() + " - " + pointer.toString().replace("~1", "/"));
+        JsonPointerLocator locator = new JsonPointerLocator("");
+        try {
+            // throw new IOException();
+            locator = new JsonPointerLocator(Files.readString(Paths.get(url)));
```
[the remark this answers is not in the document]
Comment (littlelion308) [reply src/main/java/com/ethlo/zally/ZallyRunner.java:192]: @sonatype-lift ignore
The source file is supposed to be a configurable option.

File: src/main/java/com/ethlo/zally/ZallyMojo.java [hunk 2]
```diff
@@ -285,11 +276,20 @@ private void writeResults(Map<CheckDetails, List<Result>> results)
                 printInfo("Writing result file to " + resultFile);
                 getLog().info("");
                 final Path target = Paths.get(resultFile);
```
[the remark this answers is not in the document]
Comment (littlelion308) [reply src/main/java/com/ethlo/zally/ZallyMojo.java:279]: @sonatype-lift ignore
The results file is supposed to be a configurable option.

[the remark this answers is not in the document]
Comment (littlelion308) [reply src/main/java/com/ethlo/zally/ZallyMojo.java:279]: @sonatype-lift ignore
The result file is supposed to be a configurable option.

File: src/main/java/com/ethlo/zally/ZallyMojo.java [hunk 3]
```diff
@@ -74,10 +75,10 @@ public class ZallyMojo extends AbstractMojo
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     private MavenProject project;
```
[the remark this answers is not in the document]
Comment (littlelion308) [reply src/main/java/com/ethlo/zally/ZallyMojo.java:77]: This was not effected by the changes; should it be removed anyway?

File: src/main/java/com/ethlo/zally/ZallyMojo.java [hunk 4]
```diff
@@ -74,10 +76,10 @@ public class ZallyMojo extends AbstractMojo
...
```
[the remark this answers is not in the document]
Comment (littlelion308) [reply src/main/java/com/ethlo/zally/ZallyMojo.java:77]: This was not effected by the changes; should it be removed anyway?

**Review by ethlo — APPROVED** [review]

**Comment by ethlo** [comment]
Thanks for the contribution.

**Pull request merged**